packages/a11y/src/index.js

```diff
@@ -0,0 +1,46 @@
+import addContainer from './addContainer';
+import clear from './clear';
+import domReady from '../../domReady/src/';
```

Comment:
So this is quite horrible in a way. Until `domReady` is published I cannot 
```JS
import domReady from '@wordpress/domReady';
```

Unfortunately NPM also doesn't support installing an NPM module from a github monorepo subdir [and will not in the future](https://github.com/npm/npm/issues/2974). Don't know if you can think of anything @aduth? If not, I'll just make an issue to fix this after we publish this.